DEVELOPMENT.md

```diff
@@ -47,6 +47,29 @@ Spec changes should be approved by a majority of the committers.  Approval can b
 
 No change should be approved until there is documentation for it, supplied in an accompanying PR.
 
+## Draft Features
+
+Where suitable, features will be introduced as draft but OAI approved extensions.
+By introducing new features this way we enable new features to be designed, documented and then implemented by tools that are interested in the feature, without putting the burden of implementation on all tooling.
+If the feature is successfully implemented and there is demonstrable value added by the feature, it will become a candidate for inclusion in a future release of the specification, at which point all tools will be expected to support the feature.
+
+Draft feature extensions are identified by the `x-oas-draft-` prefix and can only be used where existing extensions are permitted.
+This ensures no existing tooling will affected by the introduction of the draft feature.
+If the feature is deemed appropriate for inclusion in the OAS, the `x-oas-draft-` prefix will be removed.
+Tooling that supports draft features should plan for the future removal of the prefix.
+When tooling adds support for a later version of OAS that includes the final implementation of the feature, it MUST not support the use of the draft prefix for that feature.
+Draft features will only be promoted into minor or major releases of the specification and therefore will be transparent to OpenAPI description writers and tooling providers who choose not to use the feature while in its draft state.
+
+Draft features will be documented as GitHub issues and labeled with the `draft-feature` label.
+If during the development of a draft feature, it is determined that the feature needs to change in a way that may break existing draft implementations, the extension name itself may be versioned with a version suffix. e.g. `-v2`
+When a draft feature becomes part of a future update to the specification any version suffix will be removed.
+Draft features that are deemed not appropriate for inclusion MUST be marked with the `abandoned` label.
+Draft-features that are considered suitably specified for implementation will be marked with the `draft-ready` label.
```

Comment:
Alternative labels: "ready", "ready-to-pilot", "ready-for-trial".
"draft:feature", "draft:abandoned", "draft:ready"

Reply:
@webron and I caught up on this and have agreed on:
`draft:proposal` - we’re still hashing out the details
`draft:pilot` - go ahead and implement it
`draft:graduated` - we’ve seen the implementations, it will go into the next minor/major release
`draft:abandoned` - didn't succeed, do not use

Reply:
Latest status suggestions by @earth2marsh approved by @OAI/tsc